.github/workflows/test-zkvm.yml

```diff
@@ -149,18 +140,24 @@ jobs:
         with:
           key: rust-${{ inputs.toolchain }}-${{ inputs.zkvm }}-${{ hashFiles('Cargo.lock') }}
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
-      - name: Log in to GitHub Container Registry
-        uses: docker/login-action@v3
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Pull image ere-base-${{ inputs.zkvm }}
-        run: docker image pull ${{ needs.build_image.outputs.base_zkvm_image_tag }}
+      - name: Pull images or build locally
+        run: |
+          # Pull or build ere-base-${{ inputs.zkvm }} locally
+          if ! docker image pull ${{ needs.build_image.outputs.base_zkvm_image_tag }}; then
+            echo "Building ere-base"
+            docker build \
+              --file docker/base/Dockerfile.base \
+              --tag ${{ needs.build_image.outputs.base_image_tag }} \
+              .
+
+            echo "Building ere-base-${{ inputs.zkvm }}"
+            docker build \
+              --file docker/${{ inputs.zkvm }}/Dockerfile \
+              --tag ${{ needs.build_image.outputs.base_zkvm_image_tag }} \
+              --build-arg BASE_IMAGE_TAG=${{ needs.build_image.outputs.base_image_tag }} \
+              --build-arg CI=1 \
+              .
+          fi
```

Comment:
Use docker command to build to make sure the built images can be used in the following steps (exporting it takes very long time).